### DISCUSSION TITLE
Customizable device sets in Device Manager

### DISCUSSION THREAD

**Discussion by mattiaVDM** [responsively-org/responsively-app#862] [Ideas]
Hello,
First of all, i love the app, it is really improving my developer experience.
It would be really nice to have the feature to save custom devices sets, with an interface that could allow the user to change with one click the whole devices list. 
Lets say i define these lists of devices: 
PHONE LIST
IPhone4, iPhone5, Galaxy Fold
TABLET LIST
IPadAir, iPadMini, SurfacePro7

With a click i can switch from having PHONE LIST devices currently active in my manager, to having TABLET LIST active devices. 
I'm asking this because when i have to test different screen sizes by category (small, medium and large) i need to uncheck everything i currently have active and than search and check other devices, adding this feature could save a lot of time in the process. 
Kind regards,
Mattia

**Comment by manojVivek** [comment]
I have created #863 issue for this feature!
